src/Compilers/CSharp/Test/EndToEnd/EndToEndTests.cs

```diff
@@ -43,7 +44,17 @@ private static void RunInThread(Action action)
             }, 0);
 
             thread.Start();
-            thread.Join();
+            if (timeout is { } t && !Debugger.IsAttached)
```

Comment:
>{ } t [](http://example.com/codeflow?start=27&length=5)

nit: we don't need a second local. `is not null` or `is { }` should suffice.

Reply:
It's s struct, so I think I would then need to write `timeout.Value` or even faster `timeout.GetValueOrDefault`, so `{ } t` seems shorter.